.github/workflows/pars-upload-branch.yaml

```diff
@@ -79,3 +79,23 @@ jobs:
         with:
           name: medicines-cypress-videos
           path: medicines/pars-upload/cypress/videos
+
+      - name: DEV - Create .env file
+        working-directory: medicines/pars-upload
+        run: |
+          echo NEXT_PUBLIC_CLIENT_ID=26f95b21-63b2-475f-8a35-d39cea4cfd61 >> .env
+          echo NEXT_PUBLIC_AUTHORITY_URL=https://login.microsoftonline.com/e527ea5c-6258-4cd2-a27f-8bd237ec4c26 >> .env
+          echo NEXT_PUBLIC_REDIRECT_URI=https://mhraparsdev.azureedge.net >> .env
+          echo NEXT_PUBLIC_DISABLE_AUTH=false >> .env
```

Comment:
let's change this to

```
       run: |
          {
            echo "NEXT_PUBLIC_CLIENT_ID=26f95b21-63b2-475f-8a35-d39cea4cfd61"
            echo "NEXT_PUBLIC_AUTHORITY_URL=https://login.microsoftonline.com/e527ea5c-6258-4cd2-a27f-8bd237ec4c26"
            echo "NEXT_PUBLIC_REDIRECT_URI=https://mhraparsdev.azureedge.net"
            echo "NEXT_PUBLIC_DISABLE_AUTH=false"
          } > .env
```

Reply:
and same for the block below for Non prod (but with the non prod values

Reply:
it sucks that we're doing two `yarn export`s just because the `REDIRECT_URI` is different :/

Reply:
and client_id - but yeah, it does